conf/nebula-graphd.conf.default

```diff
@@ -23,6 +23,10 @@
 # Copy log messages at or above this level to stderr in addition to logfiles. The numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively.
 --stderrthreshold=2
 
+########## query ##########
+# Whether to treat partial success as an error
+--accept_partial_success=false
```

Comment:
Comment that it's only used for read-only access.

Reply:
Modify access don't accept partial success.

Reply:
Comments have been updated